DeveloperGame/DeveloperGame/DeveloperGameProgram.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace DeveloperGame
+{
+    class DeveloperGameProgram
+    {
+        static void Main()
+        {
+            DeveloperGameClass gameOne = new DeveloperGameClass();
```

Comment:
I don't think "gameOne" is a very good variable name

Reply:
What other names do you suggest?

Reply:
"game" or "techGame" or "quizGame" or anything else would probably be fine, but "gameOne" doesn't make much sense as it sounds like this variable will only let the user play once or something like that.